tests/error_006_import_ext_failure.ts.out

```diff
@@ -1,11 +1,11 @@
 NotFound: Cannot resolve module "./non-existent" from "[WILDCARD]/tests/error_006_import_ext_failure.ts"
+    at DenoError ([WILDCARD]/js/errors.ts:[WILDCARD])
     at maybeError ([WILDCARD]/js/errors.ts:[WILDCARD])
     at maybeThrowError ([WILDCARD]/js/errors.ts:[WILDCARD])
     at sendSync ([WILDCARD]/js/dispatch.ts:[WILDCARD])
-    at Object.codeFetch ([WILDCARD]/js/os.ts:[WILDCARD])
-    at Compiler._resolveModule ([WILDCARD]/js/compiler.ts:[WILDCARD])
+    at codeFetch ([WILDCARD]/js/os.ts:[WILDCARD])
+    at _resolveModule ([WILDCARD]/js/compiler.ts:[WILDCARD])
     at moduleNames.map.name ([WILDCARD]/js/compiler.ts:[WILDCARD])
-    at Array.map (<anonymous>)
```

Comment:
Anonymous function no longer show up?

Reply:
Yeah, sounds like something it getting dropped out of the stack when the string is generated.

Reply:
anonymous functions are not in the info provided by v8::StackTrace ... that said there's information in the structure that isn't being displaying in the current `JSError::to_string()` .. so we can probably improve the output. But AFAICS there's no way to know if there's an anonymous stack frame...

Reply:
It seems the C++ API and the JS API do not map to the same thing. Maybe we can hone these down into actual examples and send bugs to them.